package.json

```diff
@@ -10,29 +10,29 @@
     "url": "https://vufind.org/jira"
   },
   "scripts": {
-    "build": "npm run build:css",
-    "watch": "npm run watch:css",
+    "build": "npm-run-all \"build:* -- {@}\"",
```

Comment:
I'm not sure if using the wildcard approach makes sense here. The original idea was that we'd have a build:css action which could be an alias for build:scss -- this is how we transitioned from build:less to build:scss without having to revise our documentation. However, if one build task is an alias for another, running all of them is going to cause double actions.

On a related note, the existing build:scss action currently still uses grunt -- so I think maybe you want to move your work here to that action, and leave build and build:css as they are.

Of course, I'm open to a different strategy if you have a strong reason to recommend it -- just wanted to share the background behind the existing approach for context.